src/MUMPS_C-API.jl

```diff
@@ -2,21 +2,13 @@ using Libdl, LinearAlgebra, SparseArrays
 
 using MPI
 
-function __init__()
-  _DEPS_FILE = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
-  if isfile(_DEPS_FILE)
-    include(_DEPS_FILE)
-  else
-    haskey(ENV, "MUMPS_PREFIX") &&
-      error("MUMPS library not properly installed. Please run Pkg.build(\"MUMPS\")")
-  end
-end
-
-if haskey(ENV, "MUMPS_PREFIX")
-  const libsmumps = joinpath(ENV["MUMPS_PREFIX"], "lib/libsmumps.$dlext")
-  const libdmumps = joinpath(ENV["MUMPS_PREFIX"], "lib/libdmumps.$dlext")
-  const libcmumps = joinpath(ENV["MUMPS_PREFIX"], "lib/libcmumps.$dlext")
-  const libzmumps = joinpath(ENV["MUMPS_PREFIX"], "lib/libzmumps.$dlext")
+if haskey(ENV, "JULIA_MUMPS_LIBRARY_PATH")
```

Comment:
why not keep `MUMPS_PREFIX` here?

Reply:
For `QRMumps` and `NOMAD` I use the same syntax `JULIA_PACKAGE_LIBRARY_PATH`.
All JuMP solvers used this syntax too before julia v1.6 to provide the path to custom installation (`Ipopt`, `Clp`, etc...).
`Pardiso` uses `JULIA_PARDISO` instead of `JULIA_PARDISO_LIBRARY_PATH`.